src/js/media-chrome-button.js

```diff
@@ -91,30 +91,44 @@ class MediaChromeButton extends window.HTMLElement {
 
     shadow.appendChild(buttonHTML);
 
-    this.addEventListener('click', (e) => {
-      this.handleClick(e);
-    });
-
-    // NOTE: There are definitely some "false positive" cases with multi-key pressing,
-    // but this should be good enough for most use cases.
-    const keyUpHandler = (e) => {
-      const { key } = e;
-      if (!this.keysUsed.includes(key)) {
-        this.removeEventListener('keyup', keyUpHandler);
-        return;
-      }
+    this.enable();
+  }
 
-      this.handleClick(e);
-    };
+  #clickHandler = (e) => {
+    this.handleClick(e);
```

Comment:
Two functions named basically the same thing isn't great. Maybe the addition of it being private is enough.

Now I'm wondering if it'd be better to have the child class set the click handler via `super()` instead of `.handleClick`.

Reply:
renamed to `#clickListener` along with `keyupListener` and `keydownListener`, for consistency. Hopefully, makes it clear enough that there's a difference.